prow/config/jobs/test-infra.yaml

```diff
@@ -12,6 +12,17 @@ jobs:
   - name: gencheck
     command: [make, gen-check]
 
+  - name: unit-test-authentikos
+    type: presubmit
+    regex: '^authentikos/(.+\.go|go\.mod)$'
+    command: [make, -C, authentikos, unit-test]
```

Comment:
I think this is covered by `test` job already

Reply:
Because **Authentikos** is a submodule: 
#### go test ./... from the repository root will no longer test everything in the repository

https://github.com/golang/go/wiki/Modules#should-i-have-multiple-modules-in-a-single-repository

**e.g.**
```console
~/SourceCode/istio_test-infra (master)

$ go test ./...
?   	istio.io/test-infra/boskos/cmd/mason	[no test files]
?   	istio.io/test-infra/boskos/cmd/mason_client	[no test files]
ok  	istio.io/test-infra/boskos/gcp	3.122s
ok  	istio.io/test-infra/prow	(cached)
ok  	istio.io/test-infra/prow/config	1.459s
?   	istio.io/test-infra/prow/config/cmd	[no test files]
ok  	istio.io/test-infra/prow/genjobs	1.218s
?   	istio.io/test-infra/prow/genjobs/cmd/genjobs	[no test files]
ok  	istio.io/test-infra/prow/genjobs/pkg/util	(cached)
?   	istio.io/test-infra/toolbox/branch_pr	[no test files]
?   	istio.io/test-infra/toolbox/ci2gubernator	[no test files]
?   	istio.io/test-infra/toolbox/ci2gubernator/lib	[no test files]
?   	istio.io/test-infra/toolbox/deps_update	[no test files]
?   	istio.io/test-infra/toolbox/errorfinder	[no test files]
?   	istio.io/test-infra/toolbox/fixit	[no test files]
?   	istio.io/test-infra/toolbox/githubctl	[no test files]
ok  	istio.io/test-infra/toolbox/metrics	(cached)
?   	istio.io/test-infra/toolbox/metrics/buildfreshness	[no test files]
?   	istio.io/test-infra/toolbox/metrics/cmd	[no test files]
ok  	istio.io/test-infra/toolbox/metrics/coverage	(cached)
?   	istio.io/test-infra/toolbox/metrics/flakes	[no test files]
?   	istio.io/test-infra/toolbox/release_note_collector	[no test files]
?   	istio.io/test-infra/toolbox/timecomparer	[no test files]
?   	istio.io/test-infra/toolbox/util	[no test files]
?   	istio.io/test-infra/tools/github_helper	[no test files]
```